### PR TITLE
Made change to train_model to allow for changing the weight decay and…

### PR DESCRIPTION
**Pull request by MatthewPierson90**
Made change to train_model to allow for changing the weight decay and accumulate_grad_batches from the cli script.

### CODE REVIEW

**Review by jgrss — APPROVED** [review]
Thanks for these additions!

**Comment by jgrss** [comment]
@MatthewPierson90 I am finally getting around to working on `cultionet` again. Let me know if you want me to resolve the conflicts and merge this PR.

**Comment by MatthewPierson90** [comment]
Thanks Jordan, yeah resolving that conflict would be great! I believe that
the every_n_val_epochs is an older version of torch, and every_n_epochs is
current.

I got pulled away from field mapping for some time now. We have been
working on getting more training data for the model, and that has taken a
lot of time. Will definitely be excited when I get to play with the model a
bit more in the future.

Sorry for the delay.

-Matthew
